src/shared/components/ResourcesList/ResourcesList.js

```diff
@@ -67,33 +67,28 @@ ResourcesList.propTypes = {
   omitColumnsIds: PropTypes.arrayOf(PropTypes.string.isRequired),
   resourceUrlPrefix: PropTypes.string,
   disableCreate: PropTypes.bool,
-  disableEdit: PropTypes.bool,
   disableDelete: PropTypes.bool,
   disableMargin: PropTypes.bool,
   enableColumnLayout: PropTypes.bool,
   layoutNumber: PropTypes.string,
   handleRedirect: PropTypes.func,
+  filterFn: PropTypes.func,
 };
 
-ResourcesList.defaultProps = {
-  customHeaderActions: null,
-  customColumns: [],
-  createResourceForm: null,
-  showTitle: false,
-  listHeaderActions: null,
-  readOnly: false,
-  disableCreate: false,
-  disableEdit: false,
-  disableDelete: false,
-  disableMargin: false,
-  enableColumnLayout: false,
-  layoutNumber: 'StartColumn',
-  filterFn: () => true,
-};
-
-export function ResourcesList(props) {
-  const headerInjections = useGetInjections(props.resourceType, 'list-header');
-  if (!props.resourceUrl) {
+export function ResourcesList(
+  customHeaderActions = null,
+  resourceUrl,
+  resourceType,
+  resourceTitle,
+  isCompact,
+  description,
+  layoutNumber = 'StartColumn',
+  resources,
+  filterFn = () => true,
+  ...props
+) {
```

Comment:
The props should be destructured
```suggestion
export function ResourcesList({
  customHeaderActions = null,
  resourceUrl,
  resourceType,
  resourceTitle,
  isCompact,
  description,
  layoutNumber = 'StartColumn',
  resources,
  filterFn = () => true,
  ...props
}) {
```